chain/internal/tendermint/events.go

```diff
@@ -13,8 +13,8 @@ func AttributeValue(events []abcitypes.Event, eventType, attrKey string) (string
 			continue
 		}
 		for _, attr := range event.Attributes {
-			if string(attr.Key) == attrKey {
-				return string(attr.Value), true
+			if attr.Key == attrKey {
```

Comment:
Did some troubleshooting here. Printed out both sides of this "if" and it looked like this:
`attr.Key` = cGFja2V0X2RhdGE= 
`attrKey` = packet_sequence

Reply:
Nice work! What chain binaries did you use to test this? It seems tendermint changed the event attribute key to be string instead of []byte. I'm wondering if the binary used is using a tendermint v0.37-rc2?

Reply:
Hi @colin-axner. 
I was just using the `TestLearn` in examples/ibc/learn_ibc_test.go. It's using some older versions:
gaia: v7.0.0
osmosis: v11.0.0